src/store/user/provider.tsx

```diff
@@ -11,8 +11,7 @@ export const LoginProvider: React.FunctionComponent<{ children: React.ReactNode
     useEffect(() => {
         const token = localStorage.getItem("token");
         if (token) {
```

Comment:
I think, it will look cleaner if:
`if (!token) { return }
...else of the code
`